design-tokens/tokens/semantic/dimension.small.json

```diff
@@ -694,6 +694,18 @@
       "$type": "number",
       "$value": "{static.spacing.5xsmall}"
     },
+    "6xsmall": {
+      "$description": "",
+      "$extensions": {
+        "com.figma": {
+          "codeSyntax": {},
+          "hiddenFromPublishing": false,
+          "scopes": ["GAP"]
+        }
+      },
+      "$type": "number",
+      "$value": "{static.spacing.6xsmall}"
```

Comment:
For our other spacing values `3xlarge` - `4xsmall` we step down a scale step or two when in small dimension mode.

We break that pattern at `5xsmall`, presumably because only `hair` was smaller?

Additionally, the pattern for the number of steps we step down changes at `4xsmall`, `5xsmall`. Is this correct?

Should `4xsmall`, `5xsmall`, and `6xsmall` all be stepping down in size and conforming to our overall pattern?